Sources/SpeziViews/Environment/ProcessingDebounceDuration.swift

```diff
@@ -0,0 +1,29 @@
+//
+// This source file is part of the Stanford Spezi open-source project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University and the project authors (see CONTRIBUTORS.md)
+//
+// SPDX-License-Identifier: MIT
+//
+
+import SwiftUI
+
+/// An `EnvironmentKey` that provides a generalized configuration for debounce durations for any processing-related operations.
+///
+/// This might be helpful to provide extensive customization points without introducing clutter in the initializer of views.
+/// The ``AsyncButton`` is one example where this `EnvironmentKey` is used.
+public struct ProcessingDebounceDuration: EnvironmentKey {
+    public static let defaultValue: Duration = .milliseconds(150)
+}
+
```

Comment:
Same thing here, I generally add two lines between types, also applies to other parts of the PR:
```suggestion


```